src/utils.js

```diff
@@ -196,18 +196,30 @@ function getInputValue(inputElement) {
   }
 }
 
+const rolesSupportingValues = ['meter', 'progressbar', 'slider', 'spinbutton']
+function getAccessibleValue(element) {
+  if (!rolesSupportingValues.includes(element.getAttribute('role'))) {
+    return
+  }
+  // We want same behavior as accessing the `value` property
+  // eslint-disable-next-line consistent-return
+  return Number(element.getAttribute('aria-valuenow'))
+}
```

Comment:
To avoid the eslint issue, maybe you can `return undefined` inside the `if` statement.